components/home/sections/contact.js

```diff
@@ -1,98 +1,118 @@
+"use client";
 import { PYCONUG_EMAIL } from "@/utils/constants";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 
 export default function Contact() {
+  const handleFormSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      toast.success("Your email has been sent successfully 🤗", {
+        position: "bottom-center",
+        theme: "dark",
+      });
+    } catch (error) {
+      console.log(error);
+      toast.error("There was a problem sending the email", {
+        position: "bottom-center",
```

Comment:
for consistency...we could have the configurations made globally while setting  up the provider...than setting properties eg. `position` on every use case....i think we expect same position on every use case

Reply:
Ive made alll the configurations to be on the ToastContainer which i think resolves this conversation